### DISCUSSION TITLE
Dropdown menu is not hidden in Firefox on iPad

### DISCUSSION THREAD

**Discussion by andrefreitas** [community/community#68543] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,
I am using the latest version of Firefox on iPad and the Dropdown menu is not hidden by default as you can see in this screenshot:
![IMG_E158BCF14759-1](https://github.com/community/community/assets/1192089/ef3eb94f-c41a-44c0-9e27-1f5006c584e7)

Best,
André

**Comment by Cpt-KeksMonster** [comment]
Oh thanks that someone reported that. When using Safari on iPad the menu is hidden, when using Firefox 119 it is opened and cannot be closed. 😭

**Comment by i30817** [comment]
Here in a old android tablet too, but it's temporary on uncached page load and hides itself after some seconds.
